db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/SYSALIASESRowFactory.java

```diff
@@ -51,63 +51,68 @@
  *
  * Here are the directions for adding a new system supplied alias.
  * Misc:
- *  All system supplied aliases are class aliases at this point.
- *	Additional arrays will need to be added if we supply system
- *	aliases of other types.
- *	The preloadAliasIDs array is an array of hard coded UUIDs
- *	for the system supplied aliases.
- *	The preloadAliases array is the array of aliases
- *	for the system supplied aliases.  This array is in alphabetical
- *	order by package and class in Xena.  Each alias is the uppercase
- *	class name of the alias.
- *  The preloadJavaClassNames array is the array of full package.class
- *  names for the system supplied aliases.  This array is in alphabetical
- *	order by package and class in Xena.  
- *	SYSALIASES_NUM_BOOT_ROWS is the number of boot rows in sys.sysaliases
- *  in a new database.
- *
- *
+ * All system supplied aliases are class aliases at this point.
+ * Additional arrays will need to be added if we supply system
+ * aliases of other types.
+ * The preloadAliasIDs array is an array of hard coded UUIDs
+ * for the system supplied aliases.
+ * The preloadAliases array is the array of aliases
+ * for the system supplied aliases.  This array is in alphabetical
+ * order by package and class in Xena.  Each alias is the uppercase
+ * class name of the alias.
+ * The preloadJavaClassNames array is the array of full package.class
+ * names for the system supplied aliases.  This array is in alphabetical
+ * order by package and class in Xena.
+ * SYSALIASES_NUM_BOOT_ROWS is the number of boot rows in sys.sysaliases
+ * in a new database.
  */
 
-public class SYSALIASESRowFactory extends CatalogRowFactory
-{
-
-	private static final int		SYSALIASES_COLUMN_COUNT = 9;
-	private static final int		SYSALIASES_ALIASID = 1;
-	private static final int		SYSALIASES_ALIAS = 2;
-	private static final int		SYSALIASES_SCHEMAID = 3;
-	private static final int		SYSALIASES_JAVACLASSNAME = 4;
-	private static final int		SYSALIASES_ALIASTYPE = 5;
-	private static final int		SYSALIASES_NAMESPACE = 6;
-	private static final int		SYSALIASES_SYSTEMALIAS = 7;
-	public  static final int		SYSALIASES_ALIASINFO = 8;
-	private static final int		SYSALIASES_SPECIFIC_NAME = 9;
-
- 
-	protected static final int		SYSALIASES_INDEX1_ID = 0;
-
-	protected static final int		SYSALIASES_INDEX2_ID = 1;
-
-	protected static final int		SYSALIASES_INDEX3_ID = 2;
-
-	// null means all unique.
-    private	static	final	boolean[]	uniqueness = null;
-
-	private static int[][] indexColumnPositions =
-	{
-		{SYSALIASES_SCHEMAID, SYSALIASES_ALIAS, SYSALIASES_NAMESPACE},
-		{SYSALIASES_ALIASID},
-		{SYSALIASES_SCHEMAID, SYSALIASES_SPECIFIC_NAME},
-	};
-
```

Comment:
i think you wanted to move this, but it got lost :-/
see DB-10110 on master
https://github.com/splicemachine/spliceengine/pull/4129/files?diff=split&w=1#diff-20f0560c4e3b16f5e0923ce041346429fa9d9084ee732fb35e9d5afe28778dbdR122-R127

Reply:
That's right, sorry about that and thanks for spotting it.